tests/Dockerfile.centos-6

```diff
@@ -2,9 +2,8 @@ FROM centos:6
 
 # Install Ansible
 RUN yum -y update; yum clean all;
-RUN yum -y install epel-release
-RUN yum -y install git ansible sudo
-RUN yum clean all
+RUN yum -y install epel-release; yum clean all;
+RUN yum -y install git ansible sudo; yum clean all;
```

Comment:
Doesn't doing it this way eat up a lot of time with Yum doing an implicit `makecache` on every step? Instead of:

```Dockerfile
RUN yum makecache fast && \
         yum -y install deltarpm epel-release && \
         yum clean all && \
         yum -y update && \
         yum -y install git ansible sudo && \
         yum clean all
```